lua/monoglow/groups/blink-cmp.lua

```diff
@@ -0,0 +1,22 @@
+local M = {}
+
+M.url = "https://github.com/Saghen/blink.cmp"
+
+function M.get(c, opts)
+  local ret = {
+    BlinkCmpKind = { fg = c.cmp.kind },
+    BlinkCmpGhostText = { fg = c.exception },
+    BlinkCmpLabel = { fg = c.fg, bg = c.none },
+    BlinkCmpLabelMatch = { fg = c.blue2 },
+    BlinkCmpLabelDeprecated = { fg = c.cmp.deprecated, bg = c.none, strikethrough = true },
+    BlinkCmpMenu = { fg = c.comment, bg = c.none },
+    BlinkCmpMenuBorder = { fg = c.border, bg = c.none },
+    BlinkCmpDocBorder = { fg = c.border, bg = c.none },
+    BlinkCmpSignatureHelpBorder = { fg = c.border, bg = c.none }
```

Comment:
stylua adds a `,` here

```suggestion
    BlinkCmpSignatureHelpBorder = { fg = c.border, bg = c.none },
```